app/com/kinja/play/Closure.scala

```diff
@@ -523,45 +552,13 @@ object ClosureEngine {
 }
 
 /**
- * Helper object
+ * Play module.
  */
-object Closure {
```

Comment:
The methods below are now exposed by `ClosureComponent` which is essentially the "API".